recipe-server/normandy/recipes/tests/api/v2/test_api.py

```diff
@@ -546,7 +546,9 @@ def verify_signatures(self, api_client, expected_count=None):
             actual_signature = fake_sign([data])[0]['signature']
             assert actual_signature == expected_signature
 
-    def test_full_approval_flow(self, api_client):
+    def test_full_approval_flow(self, api_client, mocked_autograph):
+        # Make sure mocked_autograph fixure is included for side effects
```

Comment:
This comment could be better. Maybe something "Make sure mocked_autograph is provided so that recipes can be signed"